tests/monitoring_test.go

```diff
@@ -247,6 +247,19 @@ var _ = Describe("Prometheus Alerts", func() {
 		var createResources = func(createDataVolume bool, rxbounceEnabled bool) string {
 			vmName := fmt.Sprintf("testvmi-%v", rand.String(10))
 
+			vmi := NewMinimalVMIWithNS(strategy.GetNamespace(), vmName)
+			vm = NewVirtualMachine(vmi)
+
+			var always = kubevirtv1.RunStrategyAlways
+			vm.Spec.RunStrategy = &always
```

Comment:
Any chance to do this in a single line? E.g.,

```go
vm.Spec.RunStrategy = ptr.To(kubevirtv1.RunStrategyAlways)
```

Reply:
done